.env.development

```
@@ -10,4 +10,5 @@ NEXT_PUBLIC_KAKAO_TALK_PLUGIN_KEY=
 NEXT_PUBLIC_ADMIN_API_URL=https://api-dev.sopt.org/v2
 NEXT_PUBLIC_ADMIN_API_KEY=
 NEXT_PUBLIC_OPERATION_API_URL=https://operation.api.dev.sopt.org
-NEXT_PUBLIC_AUTH_API_URL=https://auth.api.dev.sopt.org
\ No newline at end of file
+NEXT_PUBLIC_AUTH_API_URL=https://auth.api.dev.sopt.org
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_AUTH_API_URL key should go before the NEXT_PUBLIC_DEBUG key

(UnorderedKey)